people.md

```diff
@@ -16,25 +14,17 @@ Names are followed by GitHub usernames.
 - Syed Rizvi, @srrizvi ([AMA](http://www.ama-inc.com/))
 - Andrew Cherry, @AMA-AC ([AMA](http://www.ama-inc.com/))
 - Chris Morgan, @chriscmorgan (FrontierSI)
-- Caitlin Adams (FrontierSI)
-- George Dyke, @gamedaygeorge (Secretariat member) ([Symbios](http://symbios.space))
-
-## Institutional Partners
-
-### Tier 1
-
-Institutional Council members are noted with each institution.
-
-- [Geoscience Australia](http://www.ga.gov.au/)
-- [CSIRO](https://www.csiro.au/)
-- [NASA](https://www.nasa.gov/)
-- [AMA](http://www.ama-inc.com/)
-- [USGS](https://www.usgs.gov/)
+- Caitlin Adams, @caitlinadams (FrontierSI)
+- George Dyke, @gamedaygeorge ([Symbios](http://symbios.space))
+- Stephan Meißl ([OOX](https://eox.at/))
+- Fabian Schindler, @constantinius ([OOX]())
 
```

Comment:
Still missing latest members: Tisham Dhar (Aerometrex), Robbi Bishop-Taylor (GA) - and I guess technically David Gavin (GA) too, at least for now?